docs/concepts/what-is-an-event-history.md

```diff
@@ -15,6 +15,6 @@ An append-log of [Events](/concepts/what-is-an-event) for your application.
 
 **Event History limits**
 The Temporal Cluster stores the complete Event History for the entire lifecycle of a Workflow Execution.
-There is a hard limit of 50,000 Events in a Workflow Execution Event History, as well as a hard limit of 50 MB in terms of size.
```

Comment:
This should remain unchanged. It's a different limit that happens to use the same number.